test/api/resources.js

```diff
@@ -4,8 +4,8 @@ require('dotenv').load()
 var should = require('chai').should()
 var expect = require('chai').expect
 
-var Resource = require('src/models/resource')
-var helper = require('src/helpers/resource')
+var Resource = require('../../src/models/resource')
+var helper = require('../../src/helpers/resource')
```

Comment:
Here!  With `"test": "export NODE_PATH=.:$NODE_PATH; mocha --bail --recursive"`
We can keep absolute paths thanks to `NODE_PATH=.:$NODE_PATH`, though it is a trivial change.